Doc 2807 (RC integrations) (#2940)
RedisLabs/redislabs-docs: content/rc/cloud-integrations/terraform/get-started.md

        last_four_numbers = "<Last four numbers on the card>"
    }
    ```
   

>    Example

Reviewer: ```suggestion
   Example:
```

    }
    ```

>    Example

Reviewer: ```suggestion
   Example:
```

    }
    ```
   

>    Example

Reviewer: ```suggestion
   Example:
```

3. Run `terraform apply` to apply the changes and enter `yes` to confirm when prompted.

>     This will take some time. You will see messages like this while the subscription and database are being created.

Reviewer: ```suggestion
    This will take some time. You will see messages in your terminal while the subscription and database are being created:
```

   ```text
   rediscloud_subscription.subscription-resource: Creating...
   rediscloud_subscription.subscription-resource: Still creating... [10s elapsed]
   rediscloud_subscription.subscription-resource: Still creating... [20s elapsed]
   rediscloud_subscription.subscription-resource: Still creating... [30s elapsed]
   ```

>    When complete you should see something like this:   

Reviewer: ```suggestion
   When provisioning is complete, you will see a message in your terminal:   
```

   ```text
   Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
   ```

>    If you want to verify your subscription and database creation you can view that through the Redis Cloud [admin console](https://app.redislabs.com/).

Reviewer: ```suggestion
   View the [Redis Cloud console](https://app.redislabs.com/) to verify your subscription and database creation.
```
